compiler/ast/lexer.nim

```diff
@@ -748,12 +750,12 @@ proc getEscapedChar(L: var Lexer, tok: var Token) =
 
 proc handleCRLF(L: var Lexer, pos: int): int =
   template registerLine =
-    let col = L.getColNumber(pos)
-
     when not defined(nimpretty):
-      if col > MaxLineLength:
+      if L.getColNumber(pos) > MaxLineLength:
         L.localReportPos(
           LexerReport(kind: rlexLineTooLong), pos)
+    else:
+      discard
```

Comment:
I think omitting the empty else clause in this case improves readability. Just a suggestion - no change expected from my side.

Reply:
Fair enough, removed.